extensions/kubeapi/rbac/list.go

```diff
@@ -165,3 +165,28 @@ func ListRoleTemplates(client *rancher.Client, listOpt metav1.ListOptions) (*v3.
 
 	return rtList, nil
 }
+
+// ListProjectRoleTemplateBindings is a helper function that uses the dynamic client to list projectroletemplatebindings from local cluster.
+func ListProjectRoleTemplateBindings(client *rancher.Client, listOpt metav1.ListOptions) (*v3.ProjectRoleTemplateBindingList, error) {
```

Comment:
I don't know if this would be in this PR's scope, but could you check other existing kubeapi lists (f.e. ingress list)? We had to do a refactor especially to grab names from a list. Some should have a method called `names`.

Reply:
I went through the names for ingressList. In that we are currently accepting the ingressList from the helper ListIngresses and returns a sorted list of names. In rbac, since we have different collections in the list file, we might need to add a generic function and then use that function sort the names. We will add a qa task for it and brainstorm on that part later for now. what do you think?

Reply:
Awesome, definitely agreed, I think it requires brainstorming and backlog rather than implementing it to this PR right now